mock-server/src/dals/user/user.repository.ts

```diff
@@ -14,6 +14,10 @@ export const userRepository = {
             totalPages: db.users.length,
           },
         },
+  isAlreadyEmailOnDB: async (email: string): Promise<boolean> => {
+    const doExist = db.users.find(users => users.email === email);
+    return !!doExist;
```

Comment:
here you can use some 

https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Array/some